recipe/migrations/cuda130.yaml

```diff
@@ -0,0 +1,64 @@
+migrator_ts: 1755016036
+__migrator:
+  use_local: true
```

Comment:
This should not be in the global migrator

Reply:
Removed in 5b0c52a3370e75258a60028c95658475f57f4bcf